cloudcvIde/views.py

```diff
@@ -0,0 +1,607 @@
+                            ns[blobNames[layerId]['bottom']],
+                            ns.label,
+                            include={
+                                'phase': 0
+                            })
+                    elif int(layerPhase) == 1:
+                        ns_test[topBlob] = L.SoftmaxWithLoss(
+                            ns_test[blobNames[layerId]['bottom']],
+                            ns_test.label,
+                            include={
+                                'phase': 1
+                            })
+                else:
+                    ns[topBlob] = L.SoftmaxWithLoss(
+                        ns[blobNames[layerId]['0']],
+                        ns.label)
+                    ns_test[topBlob] = L.SoftmaxWithLoss(
+                        ns_test[blobNames[layerId]['1']],
+                        ns_test.label)
+
+            elif (layerType == 'Accuracy'):
+                if layerPhase is not None:
+                    if int(layerPhase) == 0:
+                        ns[topBlob] = L.Accuracy(
+                            ns[blobNames[layerId]['bottom']],
+                            ns.label,
+                            include={
+                                'phase': 0
+                            })
+                    elif int(layerPhase) == 1:
+                        ns_test[topBlob] = L.Accuracy(
+                            ns_test[blobNames[layerId]['bottom']],
+                            ns_test.label,
+                            include={
+                                'phase': 1
+                            })
+                else:
+                    ns[topBlob] = L.Accuracy(
+                        ns[blobNames[layerId]['0']],
+                        ns.label)
+                    ns_test[topBlob] = L.Accuracy(
+                        ns_test[blobNames[layerId]['1']],
+                        ns_test.label)
+
+        train = str(ns.to_proto())
+        test = str(ns_test.to_proto())
+
+        # merge the train and test prototxt to get a single train_test prototxt
+        testIndex = [m.start() for m in re.finditer('layer', test)]
+
+        previousIndex = -1
+        for i in range(len(testIndex)):
+            if i < len(testIndex)-1:
+                layer = test[testIndex[i]:testIndex[i+1]]
+            else:
+                layer = test[testIndex[i]:]
+            a = train.find(layer)
+            if a != -1:
+                l = test[testIndex[previousIndex+1]:testIndex[i]]
+                train = train[0:a]+l+train[a:]
+                previousIndex = i
+        if previousIndex < len(testIndex)-1:
+            l = test[testIndex[previousIndex+1]:]
+            train = train + l
+
+        prototxt = train
+
+        randomId=datetime.now().strftime('%Y%m%d%H%M%S')+randomword(5)
+
+        with open(BASE_DIR+'/cloudcvIde/media/prototxt/'+randomId+'.prototxt', 'w') as f:
+            f.write(prototxt)
+
+        return HttpResponse(
```

Comment:
@gauravgupta22 since you are using django 1.9, you should use `Jsonresponse` instead of `Httpresponse`. For reference, see this https://docs.djangoproject.com/en/dev/ref/request-response/#jsonresponse-objects